src/black/__init__.py

```diff
@@ -270,7 +271,7 @@ def validate_regex(
     "-x",
     "--skip-source-first-line",
     is_flag=True,
-    help="Skip the first line of the source code.",
+    help="Skip formatting the first line of the source code.",
```

Comment:
I think the previous wording is better. We don't just skip formatting the first line; we skip even parsing it.